Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/services/GameFileCacheService.java

```diff
@@ -1,179 +1,180 @@
-package org.dolphinemu.dolphinemu.services;

```

Comment:
Seems like the reason we have such a big diff here is because this file is using CRLF in master... My bad :(